src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SuggestionModeCompletionProviderTests.cs

```diff
@@ -419,7 +419,7 @@ static void Main(string[] args)
         var x = (a$$, b)
     }
 }";
-            await VerifyBuilderAsync(markup);
+            await VerifyNotBuilderAsync(markup);
```

Comment:
Can you verify a builder here if the type isn't 'var' but is instead a good Action type?